video-app/src/app/library/store/reducers/video.reducers.ts

```diff
@@ -0,0 +1,31 @@
+import * as VideoActions from '../actions/video.actions';
+import { initializeState, VideoState } from '../states/video.state';
+import { Action, createReducer, on } from '@ngrx/store';
+
+export const initialState = initializeState();
+
+const reducer = createReducer(
+  initialState,
+  on(VideoActions.BeginYouTubeAddVideoAction, (state: VideoState, { payload }) => {

```

Comment:
`payload` to strasznie generyczna nazwa, w tym przypadku może `videoId` skoro wiesz czego tu się spodziewać?

Reply:
`=> ({  ...state, VideoId: payload, VideoError: null })` - troszkę prościej

Reply:
czy chodzio ci o to ze po zmianie payload na videoId nie trzeba robic videoId : videoId? 
rozwin co to znaczy prosciej

Reply:
zamiast robic 
``` => {
  return { ...state, VideoId: payload, VideoError: null };
}
```
możesz pominąć return i od razu zwrócić wartość:
```
=> ({ ...state, VideoId: payload, VideoError: null })
```

idealnie by było ze zmianą payload na `VideoId`:)

Reply:
zamiast robic
```
=> {
  return { ...state, VideoId: payload, VideoError: null };
}
```
możesz pominąć return i od razu zwrócić wartość:
```
=> ({ ...state, VideoId: payload, VideoError: null })
```
idealnie by było ze zmianą payload na VideoId:)